browser/ui/views/side_panel/playlist/playlist_side_panel_coordinator.cc

```diff
@@ -38,6 +42,33 @@ void PlaylistSidePanelCoordinator::CreateAndRegisterEntry(
                           base::Unretained(this))));
 }
 
+void PlaylistSidePanelCoordinator::ActivatePanel() {
```

Comment:
what happens when sidebar doesn't have playlist item? user could remove that item.

Reply:
Well, I havn't thought about it. @rebron , could you tell me what expected behavior is?

Reply:
When playlist item is removed from sidebar UI, activate playlist panel have crash.
If `SidePanelUI::GetSidePanelUIForBrowser(browser_)->Show(SidePanelEntryId::kPlaylist);` is used isntead,
crash doesn't happen but panel is not opened. It's expected behavior with latest master.

Try with this below chanage. With this, playlist panel is shown when sidebar doesn't have playlist item.
```
diff --git a/browser/ui/views/sidebar/sidebar_container_view.cc b/browser/ui/views/sidebar/sidebar_container_view.cc
index feae2a0730..4a01dfd770 100644
--- a/browser/ui/views/sidebar/sidebar_container_view.cc
+++ b/browser/ui/views/sidebar/sidebar_container_view.cc
@@ -663,9 +663,11 @@ void SidebarContainerView::OnEntryShown(SidePanelEntry* entry) {
     if (entry->key().id() == sidebar::SidePanelIdFromSideBarItem(item)) {
       auto side_bar_index = sidebar_model_->GetIndexOf(item);
       controller->ActivateItemAt(side_bar_index);
-      break;
+      return;
     }
   }
+
+  ShowSidebarAll();
 }

 void SidebarContainerView::OnEntryHidden(SidePanelEntry* entry) {


```

Reply:
Cool, changed and rebased.